extension/README.md.args.mjs

```diff
@@ -34,22 +34,26 @@ yarn chain
 > in a second terminal window, 🛰 deploy your contract (locally):
 
 \`\`\`sh
-cd <challenge_folder_name>
 yarn deploy
 \`\`\`
 
 > in a third terminal window, start your 📱 frontend:
 
 \`\`\`sh
-cd <challenge_folder_name>
 yarn start
 \`\`\`
 
 📱 Open http://localhost:3000 to see the app.
 
 > 👩‍💻 Rerun \`yarn deploy --reset\` whenever you want to deploy new contracts to the frontend, update your current contracts with changes, or re-deploy it to get a fresh contract address.
 
-🔏 Now you are ready to edit your smart contract \`{mainContractName.sol}\` in \`packages/hardhat/contracts\`
+---
+
+⚠️ We have disabled AI in Cursor and VSCode and highly suggest that you do not enable it so you can focus on the challenge, do everything by yourself, and hence better understand and remember things. If you are using another IDE, please disable AI yourself.
+
+🔧 In case you want to enable AI for some reason, you can do it in the root folder of the project.
```

Comment:
Something I have been thinking about a lot recently is how are we making our tooling accessible to "vibe-coders" and disabling AI makes it very inaccessible. This is still a good idea for the 99% of builders who need to understand the code fundamentally but perhaps we could call out the vibe-coders in this sentence by changing it to this:
"If you are a vibe-coder and don't care about understanding the syntax of the code used and just want to understand the general takeaways, you can re-enable AI by ..."

I think this does two things:
- Makes vibe-coders feel seen
- Makes it clear that if they do this they are sacrificing the ability to understand the code

Reply:
Good point, thanks! [42f293b](https://github.com/scaffold-eth/se-2-challenges/pull/368/commits/42f293b22499cc0379674a21570c0e63f2be8bff)